pydicom/contrib/imViewer_Simple.py

```diff
@@ -234,25 +263,49 @@ def get_LUT_value(self, data, window, level):
     # -----------------------------------------------------------
     def loadPIL_LUT(self, dataset):
         if not have_PIL:
-            raise ImportError("Python Imaging Library is not available. See http://www.pythonware.com/products/pil/ to download and install")
+            raise ImportError("Python Imaging Library is not available."
+                              " See http://www.pythonware.com/products/pil/"
+                              " to download and install")
         if('PixelData' not in dataset):
-            raise TypeError("Cannot show image -- DICOM dataset does not have pixel data")
-        if('WindowWidth' not in dataset) or ('WindowCenter' not in dataset):  # can only apply LUT if these values exist
+            raise TypeError("Cannot show image -- "
+                            "DICOM dataset does not have pixel data")
+
+        # can only apply LUT if these values exist
+        if('WindowWidth' not in dataset) or ('WindowCenter' not in dataset):
             bits = dataset.BitsAllocated
```

Comment:
I didn't edit this bit I don't think (meaning it didn't have an exception previously). Would you like me to add? This is also a contributed script, in case we need feedback from the author on other things.